app-misc/cmatrix/cmatrix-2.0-r5.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+FONT_SUFFIX="pcf"
+
+inherit cmake font
+
+DESCRIPTION="Ncurses based app to show a scrolling screen from the Matrix"
+HOMEPAGE="https://github.com/abishekvashok/cmatrix"
+SRC_URI="https://github.com/abishekvashok/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
```

Comment:
this should be `GPL-3+`, see https://github.com/abishekvashok/cmatrix/blob/master/cmatrix.c#L9-L12

Reply:
Good call. Also they got conflicting headers and [readme](https://github.com/abishekvashok/cmatrix?tab=readme-ov-file#page_facing_up-license) license